lib/fast_gettext/translation_repository/yaml.rb

```diff
@@ -38,8 +38,8 @@ def reload
       def find_and_store_files(options)
         @files = {}
         path = options[:path] || 'config/locales'
-        Dir["#{path}/??.yml"].each do |yaml_file|
-          locale = yaml_file.match(/([a-z]{2})\.yml$/)[1]
+        Dir["#{path}/*.yml"].each do |yaml_file|
+          locale = yaml_file.match(/([a-zA-Z_]*)\.yml$/)[1]
```

Comment:
how about `File.basename(yaml_file, '.yml')` ?

Reply:
a test with such a file would be nice too

Reply:
For sure. I just left work but I can look into adding it soon!

Reply:
What is the purpose of the `not_found.yml` file? Unfortunately, this fix will allow that to pass this `Dir...` check.

Reply:
try a `git blame` ... idk why it is there ... afaik can be removed ... seems weird that someone would add random .yml files into that folder anyway

Reply:
Cool cool. Just wanted to check in case before proceeding. Thanks!